### ISSUE TITLE
Can't compile

### ISSUE THREAD

**Issue by Lahiri** [nyash-qq/galaxy-plugin-twitch#5]
I have not been able to install this integration because when I run 
`inv install`
I get these errors:

`Traceback (most recent call last):
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python36-32\Scripts\inv.exe\__main__.py", line 9, in <module>
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\site-packages\invoke\program.py", line 373, in run
    self.parse_collection()
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\site-packages\invoke\program.py", line 465, in parse_collection
    self.load_collection()
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\site-packages\invoke\program.py", line 696, in load_collection
    module, parent = loader.load(coll_name)
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\site-packages\invoke\loader.py", line 76, in load
    module = imp.load_module(name, fd, path, desc)
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "c:\users\myuser\appdata\local\programs\python\python36-32\lib\imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\myuser\AppData\Local\GOG.com\Galaxy\plugins\installed\galaxy-plugin-twitch\tasks.py", line 8, in <module>
    from galaxy.tools import zip_folder_to_file
ModuleNotFoundError: No module named 'galaxy'`

Forgive me if it's something obvious, but I am not familiar with compiling python.
I have python 3.6.4 installed on my system.
Thanks

**Comment by Danyelalejandro** [comment]
@Lahiri use instead the official release at https://github.com/nyash-qq/galaxy-plugin-twitch/releases. Download the most recent one, unpack the zip and copy the folder to %localappdata%\GOG.com\Galaxy\plugins\installed
It will work

**Comment by Lahiri** [comment]
@Danyelalejandro thanks!

**Comment by Rall3n** [comment]
Did you run `pip install -r requirements.txt` before using `inv install`?
`ModuleNotFoundError'` means that it could not find a package that should have been installed before using `inv install`.

Also it would be wise not to use clone the repository into the `install` folder for Galaxy 2.0 plugins. The source files would be overwritten and the script would end in an error.

@nyash-qq I think the `README.md` should be updated to use `pip install -r requirements.txt` before `inv install` to make sure the `galaxy` module is installed.

**Comment by Lahiri** [comment]
@Rall3n No I did not. I just run the code that is indicated in the readme, so: 
`
git clone https://github.com/nyash-qq/galaxy-plugin-twitch.git
cd galaxy-plugin-twitch
pip install invoke
inv install
`

I actually used a different folder from the GOG Galaxy installed folder to begin with, but since it wasn't working I tried to move it there, with no changes in behavior.

I am now using the release as suggested by Danyelalejandro.
